src/commands/flutter-symbols/__tests__/fixtures/pubspecs/invalidPubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: invalid_pubspec_example
+description: A pubspec that's just slightly off for testing
```

Comment:
Maybe we could specify what's off with this pubspec so that it makes it easier to understand what's supposed to be detected